server/test/handlers/getWorkersForSchool.test.js

```diff
@@ -5,130 +5,131 @@ describe('testing user to input schoolID to see worker ID ', () => {
-
-// Assert
-//expect(workerIds).toStrictEqual(workerIds);
-//});
-
-//test('rejection of empty string value for required schoolId parameter', async () => {
-// Arrange
-//const testschoolId = '';
-
-// Act
-//const workerIds = await schoolHandler.getWorkerIdsForSchool(testschoolId);
-//const workerIds = await schoolHandler.getWorkerIdsForSchool();
-
-// Assert
-//expect(workerIds).toStrictEqual(workerIds);
-//});
-
-//test('rejection of invalid school id', async () => {
-// Arrange
-//const testschoolId = 0;
-
-// Act
-//const workerIds = await schoolHandler.getWorkerIdsForSchool(testschoolId);
-//const workerIds = await schoolHandler.getWorkerIdsForSchool();
-
-// Assert
-//expect(workerIds).toStrictEqual(workerIds);
-//});
-//});
-
-//describe('testing valid school ID parameters', () => {
-//beforeEach(() => {
-//jest.resetModules(); // Clears any cache between tests.
-//});
-
-//test('empty strings for optional fields', async () => {
-// Arrange
-//const testschoolId = 1;
-
-// Act
-//const workerIds = await schoolHandler.getWorkerIdsForSchool(testschoolId);
-//const workerIds = await schoolHandler.getWorkerIdsForSchool();
-
-// Assert
-//expect(workerIds).toStrictEqual(workerIds);
-//});
-
-//test('null values for optional fields', async () => {
-// Arrange
-//const testschoolId = 1;
-
-// Act
-//const workerIds = await schoolHandler.getWorkerIdsForSchool(testschoolId);
-//const workerIds = await schoolHandler.getWorkerIdsForSchool();
-
-// Assert
-//expect(workerIds).toStrictEqual(workerIds);
-//});
-
-//test('only passing in values for required fields', async () => {
-// Arrange
-//const testschoolId = 1;
-
-// Act
-//const workerIds = await schoolHandler.getWorkerIdsForSchool(testschoolId);
-//const workerIds = await schoolHandler.getWorkerIdsForSchool();
-
-// Assert
-//expect(workerIds).toStrictEqual(workerIds);
-//});
-
-//test('passing in valid string values for required fields', async () => {
-// Arrange
-//const testschoolId = "1";
-
-// Act
-//const workerIds = await schoolHandler.getWorkerIdsForSchool(testschoolId);
-//const workerIds = await schoolHandler.getWorkerIdsForSchool();
-
-// Assert
-//expect(workerIds).toStrictEqual(workerIds);
-//});
-
-//test('passing in valid values for all fields', async () => {
-// Arrange
-//const testschoolId = 1;
-
-// Act
-//const workerIds = await schoolHandler.getWorkerIdsForSchool(testschoolId);
-//const workerIds = await schoolHandler.getWorkerIdsForSchool();
-
-// Assert
-//expect(workerIds).toStrictEqual(workerIds);
-    //});
-});
\ No newline at end of file
+        // Assert
+        expect(workerIds).toStrictEqual(workerIds);
+        //});
+
+        test('passing in valid string values for required fields', async () => {
+            // Arrange
+            const testschoolId = "1";
+
+            // Act
+            const workerIds = await schoolHandler.getWorkerIdsForSchool(testschoolId);
+            const workerIds = await schoolHandler.getWorkerIdsForSchool();
```

Comment:
Hi Simran,
I am confused by why the method is called twice in the same unit test and stored in the same variable. May you please clarify the purpose behind this duplication?